CONTRIBUTING.md

```diff
@@ -46,3 +46,13 @@ The outline above is an extremely simplified version. In real life you have to d
 - Make sure test-suite passes: `npm test`
 - Squash your commits
 - Write a convincing description of your PR and why we should land it
+
+## Release Checklist
+
+- Run [`npm version patch`](https://docs.npmjs.com/cli/version) to update the version number. Use `minor` or `major` instead of `patch` if needed (see [semver.org](http://semver.org/) for details).
+- Open a [pull request](https://github.com/OctoLinker/browser-extension/pulls) with the new version.
+- Once the pull request is merged in, tag the resulting commit as `vX.Y.Z` (where `X`, `Y`, `Z` are the major, minor, and patch versions).
+- Push the tag to GitHub. This will trigger Travis CI to create a new [GitHub Release](https://github.com/OctoLinker/browser-extension/releases) and submit the new Chrome extension to the Chrome Web Store. See [.travis.yml](https://github.com/OctoLinker/browser-extension/blob/master/.travis.yml) for details.
+- Submit `firefox-octolinker-X.Y.Z.zip` from the [GitHub Release](https://github.com/OctoLinker/browser-extension/releases) to [addons.mozilla.org](https://addons.mozilla.org/en-US/developers/addon/octolinker/versions#version-upload). Be sure to include the `Source code (zip)` file from the release as well.
+- Submit `opera-octolinker-X.Y.Z.zip` from the [GitHub Release](https://github.com/OctoLinker/browser-extension/releases) to [addons.opera.com](https://addons.opera.com/developer/package/226344/?tab=versions). Afterwards, go to the [Conversation tab](https://addons.opera.com/developer/package/226344/?tab=conversation), add a link to the `Source code (zip)` file and copy/paste the build instructions from previous releases.
+- Update release notes at https://github.com/OctoLinker/browser-extension/releases/tag/vX.Y.Z
```

Comment:
Hmm, but doesn't Travis just create a new release alongside the draft? Then the draft would have to be deleted, after copying its body to the Travis-created release.

I came across a tool which can help gather and format changes since the last release: [`github_changelog_generator`](https://github.com/skywinder/github-changelog-generator). Maybe we can find a way to integrate that into the release process? For now, though, I'll update the proposed checklist with a [`{LAST_VERSION}...master
`](https://github.com/OctoLinker/browser-extension/compare/%7BLAST_VERSION%7D...master) link.